### ISSUE TITLE
[Bug]Filter Category

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#1436]
when filtering category (in this case Crafting Formulas) and the Formula has a diffrent Category than the Item, Formula wont show up in list

https://gyazo.com/cc9758fad880e5ab4e572558f7779a7b

**Comment by insthync** [comment]
I will change condition to
```
                if ((string.IsNullOrEmpty(entry.Category) && string.IsNullOrEmpty(entry.ItemCraft.CraftingItem.Category)) ||
                    filterCategories.Contains(entry.Category.Trim().ToLower()) ||
                    filterCategories.Contains(entry.ItemCraft.CraftingItem.Category.Trim().ToLower()))
                {
                    // Category filtering
                    result.Add(entry);
                }
```

**Comment by insthync** [comment]
Fixed in 1.73b
